### ISSUE TITLE
Modify code to meet accessibility requirements

### ISSUE THREAD

**Issue by karlnh** [karlnh/module-1-challenge#5]
# Introductory links
[Click here for articles on how to address accessibility.](https://developer.mozilla.org/en-US/docs/Web/Accessibility)
[Click here for Web Content Accessibility Guidelines (WCAG) 2.1 at a Glance,](https://www.w3.org/WAI/standards-guidelines/wcag/glance/)

# HTML
Semantic HTML tags help screenreaders navigate and control a page in predictable and useful ways. Moreover, semantic tags come with certain functions and stylings, and are easier to understand. They work well on mobile and improve search engine results.

## Using clear language
Concise and unambiguous language is important for screenreaders to accurately relay the meaning of text. This means generally avoiding contractions of any kind:
- Avoid symbols in favor of text when possible; "5 to 7" instead of "5-7".
- Expand abbreviations; "January" instead of "Jan".
- Introduce acronyms in their expanded form and use the _abbr_ tag to describe them later on.

## Text alternatives
It is important to provide text alternatives to non-text media. Alt text should describe the image in a way that makes the image's importance clear. Is the image purely decorative or does it show something meaningful? The text should not contain redundant information but still fully describe relevant details.
- Extra contextual information can be provided with the _title_ attribute.

# CSS
Like HTML, you should use logical and legible design. Headings should stand out from bodies; lists should have distinct elements. Text should contrast with the background [click here to view a page on styling text in CSS](https://developer.mozilla.org/en-US/docs/Learn/CSS/Styling_text).

## Changing colors
- HTML uses both _strong_ and _em_ to place emphasis on text. One is encouraged to add some slight coloring to these tags in CSS.
- The same can be done for the _abbr_ tag.

## Link styling for CSS
- It is important that there is a change in color associated with change in link state, i.e. a link is by default one color and changes to another color when clicked. Feel free to play around with color as long as it contrasts with the background.
- Link text color (like any other text) must have a 4.5:1 contrast from the background color, and also different from non-linking text (3:1 contrast). There are websites that will help you affirm sufficient contrast between colors. [Click here to visit one.](https://webaim.org/resources/contrastchecker/)

**Comment by karlnh** [comment]
**Accessibility:**
- Header links should change color on hover.
- Search Engine Optimization (SEO) should take an _abbr_ tag.
- All images should be provided with alt text describing their relevance.
- Replace div tags with semantic tags as necessary.

**Cleanup:**
- Again, replacing div tags. Formatting also.
- Merge some selectors that make use of redunant elements (a few images can share the same max-height)
- Do some slight color changes on unique text styles.
- Maybe put the font-family in the universal selector and styling selectors as necessary?
- Maybe mess with link styling?

**Comment by karlnh** [comment]
This was way too extra. I didn't have to do like 99% of this. But it was good to learn about!
